src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -63,14 +105,14 @@ class CallActor(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backe
 
     def failedInstantiation(e: Throwable): Unit = {
       val message = s"Call '${call.fullyQualifiedName}' failed to launch command: " + e.getMessage
-      log.error(e, s"$tag: $message")
+      log.error(e, s"$tag: $call failed: $message")
       context.parent ! CallFailed(call, message)
     }
 
     def launchCall(commandLine: String): Unit = {
-      log.info(s"$tag: launching `$commandLine`")
+      log.info(s"$tag: launching ${call.name} for workflow ${workflowDescriptor.name} as `$commandLine`")
```

Comment:
Workflow UUID should definitely be in the log message somewhere because `workflowDescriptor.name` could be ambiguous.  Two workflows with the same name can run simultaneously.

Also, keeping this log message short is important because the command lines tend to be long so leaving it a lot of room to grow is nice.